src/components/Dashboard/PaperFiling.vue

```diff
@@ -4,7 +4,8 @@
       as paper documents.</p>
     <p>To request copies of paper documents, contact BC Registry Staff with the document you require and
       the name and incorporation number of your association:</p>
-    <ErrorContact class="mt-5" />
+
+    <ErrorContact class="pt-3" />
```

Comment:
I finally adopted this approach of using helper classes for padding and margins after seeing you use them, so useful! Thanks for using them consistently 👍

Reply:
Cool. My approach is:
- I think it usually makes sense to use spacing/layout helper classes in the template, where you can see how the different components are laid out (instead of hiding these styles in a CSS block).
- However, elements that need common (or specific) styling should use specific class names.

In general I think it's good to try to keep "layout" separate from "styling".

Reply:
Great points and a good rule of thumb to follow 👍